pages/real_time.py

```diff
@@ -0,0 +1,303 @@
+import dash
+from dash import Dash, dcc, html, Input, Output, State, dash_table, callback
+import dash_bootstrap_components as dbc
+
+import plotly.express as px
+import plotly.graph_objs as go
+
+import pandas as pd
+
+from collections import deque
+
+import subprocess
+
+import time
+import json
+
+from utils import log_example
+from utils import transformation
+
+
+processing_job_name = None
+
+# Cmd line for process
+process = ["python", "json_generator.py"]
+
+# Data for asset dropdown menu
+asset_options = []
+with open('asset.txt', 'r') as f:
```

Comment:
Avoid writing the code in the global context. Assign only constants here or invoke some function or methods. Create  a separate method for reading asset or use OOP approach and put such things in __init__